openjdk13/plan.sh

```diff
@@ -0,0 +1,62 @@
+pkg_origin=core
+pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
+pkg_name=openjdk13
+pkg_build_version=33
+pkg_build_ref=5b8a42f3905b406298b72d750b6919f6
+pkg_version=13.0.0
+pkg_filename=openjdk-13_linux-x64_bin.tar.gz
+pkg_source="https://download.java.net/java/GA/jdk13/${pkg_build_ref}/${pkg_build_version}/GPL/${pkg_filename}"
+pkg_shasum=5f547b8f0ffa7da517223f6f929a5055d749776b1878ccedbd6cc1334f4d6f4d
+pkg_dirname='jdk-13'
+pkg_license=('GPL-2.0-with-classpath-exception')
+pkg_description=('GNU General Public License, version 2, with the Classpath Exception build of OpenJDK13')
+pkg_upstream_url='https://jdk.java.net/13/'
+pkg_deps=(
+  core/alsa-lib
+  core/freetype
+  core/gcc-libs
+  core/glibc
+  core/libxext
+  core/libxi
+  core/libxrender
+  core/libxtst
+  core/xlib
+  core/zlib
+)
+pkg_build_deps=(core/patchelf core/rsync)
+pkg_bin_dirs=(bin)
+pkg_lib_dirs=(lib)
+pkg_include_dirs=(include)
+
+source_dir=$HAB_CACHE_SRC_PATH/${pkg_dirname}
+
+do_setup_environment() {
+ set_runtime_env JAVA_HOME "$pkg_prefix"
+}
+
+do_build() {
+  :
```

Comment:
hrmm `:`? is this the same as `return 0`?

Reply:
I'm pretty sure it evaluates to `true`. But this should be changed to `return 0`.

Reply:
Why? It's the same thing. `:` and `true` are both POSIX compliant. In bourne shells `true` is often a shell script of `:`.

Reply:
It is a matter of standards in core plans. We use `return 0`, and this is what is documented for habitat plans. And from my personal perspective, I feel that `return 0` is more friendly to read.

Reply:
To add clarity, I would recommend `return 0`.

Reply:
I changed the functions. Is there a style guide or plan linter that I missed?

Reply:
Not that I am aware of, but that could be a fun project ;)